node/Main.hs

```diff
@@ -68,7 +69,15 @@ action (SimpleNodeArgs (cArgs@CommonNodeArgs {..}) (nArgs@NodeArgs {..})) = do
     let vssSK = fromJust $ npUserSecret currentParams ^. usVss
     let gtParams = CLI.gtSscParams cArgs vssSK (npBehaviorConfig currentParams)
 
-    case sscAlgo of
+    let onNtpStatus = \case
+            NtpSyncOk -> logDebug $
+                      -- putText  $ -- FIXME: for some reason this message isn't printed
```

Comment:
AFAIU, that's due to yet another bug in log-warper which people are resolving, is it correct?

Reply:
if I knew I'd reference the relevant issue id